### DISCUSSION TITLE
Configuration for Editing of Questions during review of question by Question paper creator (Sourcing reviewer)

### DISCUSSION THREAD

**Discussion by vinod303** [Sunbird-coKreat/Community#24] [General Discussions]
Ref. https://project-sunbird.atlassian.net/browse/CO-61
@tarang27  , @Prerna789  , with regard to above ticket, please clarify - does this feature require a config change? In which case , please provide the required configuration details.
cc: @pallakartheekreddy  @ashokreddy1208  @amitpriyadarshi

**Comment by vinod303** [comment]
@tarang27 , @Prerna789 ,  can u please respond to the above query
cc: @pallakartheekreddy @ashokreddy1208 @amitpriyadarshi

**Reply by tarang27** [reply]
@vinod303 updated the ticket with the config
